README.md

```diff
@@ -83,7 +83,7 @@ The jsonb WAL record is in the following format for inserts and updates.
     ]
 }
 ```
-where `action` may be `I` or `U` for insert, updated, and delete respectively.
+where `action` may be `I`, `U` or `D` for insert, updated, and delete respectively.
```

Comment:
@dshukertjr thanks for taking a look at this! I think we meant to take out delete from this section b/c it's covered below. The reason being it differs from inserts and updates. I'll leave it up to @olirice though!

```suggestion
where `action` may be `I` or `U` for insert and updated, respectively.
```

Reply:
Thanks for such a quick response!